samples/Samples.AspNetMvc5/Samples.AspNetMvc5.csproj

```diff
@@ -4,7 +4,7 @@
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <Platform Condition=" '$(Platform)' == '' ">x64</Platform>
```

Comment:
Is it an issue that this block doesn't have an equivalent for `x86`?

Reply:
Missed the comment in the PR description about setting IISExpress to 64bit only.  Why don't we also need an x86 configuration for for IIExpress?

Reply:
These lines set the defaults values for the `Configuration` and `Platform` properties if they are not set when building. It's basically:
```
if(Configuration == "") { Configuration = "Debug" }
if(Platform == "") { Platform = "x64" }
```
The defaults are used if you run `msbuild` without specifying values for these properties, for example.

This is different than `<Platforms>x64;x86</Platforms>` (plural) in `Samples.AspNetCoreMvc2.csproj`, which indicates which platforms are _available_ to build and is used, for example, in IDEs to show a drop-down list of available platforms. If not specified, the default list is `AnyCPU`, which I wanted to remove.

(btw, `AspNetCoreMvc2` uses the newer so-called "SDK" format for the `csproj` file, since it needs to support .NET Core in addition to .NET Framework.)

Reply:
re: IIS Express 64

This only applies when running the app from Visual Studio. It's always going to run either the 32-bit version or the 64-bit version of IIS Express. If a new developer grabs this repo and tries to run one of the sample apps, I want everything to default to 64-bits (native dlls + sample apps + IIS Express), since 64-bits is the main use case. If needed, the developer can switch all the builds to 32-bits from a single drop-down in the Visual Studio UI. For IIS Express, they need to go into the sample project's properties. It's a better default than _always_ having to go and change IIS Express to 64-bits.

The worst part about all this is that if you don't set them correctly (e.g. 64-bit native dll with a 32-bit IIS Express and app), the profiler can't hook into the process and it all just fails silently. Happens to me all the time.